satviz-consumer/src/main/java/edu/kit/satviz/consumer/config/HeatmapColors.java

```diff
@@ -74,11 +75,11 @@ public boolean equals(Object o) {
       return false;
     }
     HeatmapColors colors = (HeatmapColors) o;
-    return fromColor == colors.fromColor && toColor == colors.toColor;
+    return hotColor == colors.hotColor && coldColor == colors.coldColor;
```

Comment:
Da wir hier keine primitiven Datentypen mehr verwenden, sollte hier "equals" verwendet werden.
```suggestion
    return coldColor.equals(colors.coldColor) && hotColor.equals(colors.hotColor);
```